end-to-end-test/remote/specs/core/comparisonTab.spec.js

```diff
@@ -1,135 +1,179 @@
 var assert = require('assert');
-var goToUrlAndSetLocalStorage = require('../../../shared/specUtils')
-    .goToUrlAndSetLocalStorage;
-var { jsApiClick } = require('../../../shared/specUtils');
+const asyncUtils = require('../../../shared/specUtils_Async');
+
+const { jsApiClick, getElementByTestHandle, getElement } = asyncUtils;
 
 const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, '');
-const SampleCreateGroupButton =
-    'button[data-test="sampleGroupComparisonCreateGroupButton"]';
-const PatientCreateGroupButton =
-    'button[data-test="patientGroupComparisonCreateGroupButton"]';
+const getSampleCreateGroupButton = async function() {
```

Comment:
i don't think you need to wrap these in fuctions.  enough to use getElementByTestHandle directly.